_digest/october-2020/incorporate-social-life-in-student-life.mdx

```diff
@@ -0,0 +1,60 @@
+date: 10/04/2020
+releaseBatch: October 2020
+summary: As a high school student myself, I know firsthand just how busy our daily schedules can get. Between all the homework, volunteering, part-time jobs, and extracurricular activities, there is barely time where you can just be a teenager, let alone go to wild ragers every other night. I understand how difficult it can be to maintain a social life while also keeping up with all of your academic commitments, so here are 4 tips to help you accomplish just that. 
+authors: ["Phoebe Yu"]
+tags: ["Productivity", "Student Life"]
+---
+<div>
+	
+<p> High school. An incredibly memorable, hectic, and exhilarating 4 years of your life. According to all those cheesy high school movies, it is a time when you go to parties every other night, when you ditch class to go on wild adventures with your friends, and when everyone in the cafeteria can simultaneously burst into song and dance… It doesn’t take a genius to realize that high school is nothing like how it’s portrayed in the movies. As a high school student myself, I know firsthand just how busy our daily schedules can get. Between all the homework, volunteering, part-time jobs, and extracurricular activities, there is barely time where you can just be a teenager, let alone go to wild ragers every other night. I understand how difficult it can be to maintain a social life while also keeping up with all of your academic commitments, so here are 4 tips to help you accomplish just that. </p>
+
+<ol>
+    <li> <strong>Do Homework With Peers</strong> </li>
+</ol>
+
+<p> Doing homework may not be the most fun activity in the world, but it is an important part of learning nonetheless. One way to make studying more enjoyable is to do it with your friends. There are many ways you can do this. For example, you could set up <strong>study groups</strong> or <strong>video chat </strong>each other while doing homework. Studying together is not only fun, but can be beneficial. You and your friends can share ideas, help each other out with difficult concepts, and provide new perspectives on certain problems. As long as you and your friends stay focused, this is a great way to socialize while also upholding your responsibilities. </p>
+
+
+<ol start="2">
+    <li> <strong>Participate in Extracurriculars Together</strong> </li>
+</ol>
+
+<p> Most high school students have commitments outside of school, whether it be volunteering, sports teams, clubs, or non-profit organizations. These commitments can take up quite a lot of time especially if you are dedicated. If your friends have similar interests, it would be a good idea to participate in these extracurriculars together. Not only would it make the experience more enjoyable, but it’s also a great way to<strong> bond while doing something you are all passionate about</strong>. </p>
+
+
+<ol start="3">
+    <li> <strong>Find a Job With Friends</strong> </li>
+</ol>
```

Comment:
I don't think you need to do this, either make it all one ordered list (like in the comment on the `5 jobs in medical field` article) 
```
<ol> 
  <li>...</li> 
  <p> ... </p> 
  <li>...</li> 
  <p> ... </p> 
</ol> 
```
or just use a h2 or h3 with the number item `<h3>3. Find a Job with Friends</h3>`